lib/smile-identity-core/web_api.rb

```diff
@@ -91,7 +91,7 @@ def id_info=(id_info)
       # if it's a boolean
       updated_id_info[:entered] = id_info[:entered].to_s if !updated_id_info[:entered].nil? == updated_id_info[:entered]
 
-      if updated_id_info[:entered] && updated_id_info[:entered] == 'true'
+      if updated_id_info[:entered] && updated_id_info[:entered] == 'true' && @partner_params[:job_type].to_i != JobType::DOCUMENT_VERIFICATION
```

Comment:
You should check for equality instead. `!= JobType::DOCUMENT_VERIFICATION` will be more fragile than checking against known job types. `[country id_type id_number]` are required for `JobType::BIOMETRIC_KYC`, `JobType::DOCUMENT_VERIFICATION` needs `[country]`

Reply:
I'm not sure i get the equality part. I'm not checking the type here. `JobType::DOCUMENT_VERIFICATION` resolves to `6`. See https://github.com/smileidentity/smile-identity-core-ruby/blob/main/lib/smile-identity-core/constants/job_type.rb#L17